docs/nuget-org/Deprecate-packages.md

```diff
@@ -61,3 +61,82 @@ Project `My.Test.Project` has the following deprecated packages
+
+Package authors who have deprecated a legacy package can choose to transfer its "popularity" to a newer package to boost the newer package's search ranking. This helps customers discover the newer package instead of the deprecated package.
+
+For example, let's say I have two packages:
+
+* My deprecated legacy package, `Contoso.Legacy` with 3 million downloads
+* My latest package, `Contoso.Latest` with 5 downloads
+
+NuGet.org prefers search results with higher downloads/popularity. Given the search query "Contoso", my deprecated package `Contoso.Legacy` would likely rank above my latest package `Contoso.Latest` in search results.
+
+To solve this problem, I can apply to transfer the popularity of my deprecated legacy package to my latest package. This would cause `Contoso.Latest` to rank higher in search results, while `Contoso.Legacy` would rank lower. Only the internal popularity scores for the packages is impacted, the actual download count for each package will not be affected.
+
+> [!Note]
+> Popularity transfers can make it significantly harder for consumers to find the legacy package.
+
+See the table below to get a concrete idea of how a popularity transfer may impact search rankings for the query "Contoso":
+
+| Search ranking 	| Before popularity transfer     	| After popularity transfer      	|
+|----------------	|--------------------------------	|--------------------------------	|
+| 1              	| *Contoso.Legacy, 3M downloads*  	| *Contoso.Latest, 5 downloads* 	|
+| 2              	| Contoso.Scanner, 2M downloads  	| Contoso.Scanner, 2M downloads  	|
+| 3              	| Contoso.Core,  1.5M downloads  	| Contoso.Core,  1.5M downloads  	|
+| 4              	| Contoso.UI, 1M downloads       	| Contoso.UI, 1M downloads       	|
+| ...            	| ...                            	| ...                            	|
+| 20             	| *Contoso.Latest, 5 downloads* 	| *Contoso.Legacy, 3M downloads*   	|
+
+### Popularity transfer application process
+
+1. Review the [popularity transfer requirements](#popularity-transfer-requirements).
+2. Email [account@nuget.org](mailto:account@nuget.org) with the deprecated package whose popularity should be transferred, and, the list of stable package(s) that should receive the popularity transfer.
+
+After the application is submitted, we will notify you of your application's acceptance or rejection (with the criteria that caused rejection). We may need to ask additional identifying questions to confirm owner identity.
+
+#### Popularity transfer requirements
+
+* The legacy packages and new packages must share all owners.
+* The new packages must be clearly related to the legacy packages in naming and function (i.e. an evolution or next generation).
+* All versions of the legacy packages must be deprecated and point to the new packages receiving the transfer.
+* The popularity transfer must not cause confusion for NuGet users or worsen the NuGet search experience.
```

Comment:
This stands out as important to me. Looking at the table above, I can see users being confused by their search results because the actual download counts don't match the order of popularity, leading to mistrust in the nuget.org search results.

Do we have any way of communicating to users that a package's popularity has been transferred - or that a package is special/an outlier in some way? I'm thinking in a subtle, visual way that can be seen at a glance on the search results.

Reply:
> Looking at the table above, I can see users being confused by their search results because the actual download counts don't match the order of popularity, leading to mistrust in the nuget.org search results.

This is an excellent question. Keep in mind that search rankings tries to find results that are popular *and* relevant to the query. In other words, it's totally fine for the top results to have low downloads as long as they are the best matches for the search query. If we simply ordered results by their downloads, packages with many downloads would always drown out packages that are most relevant to the user's query. 

We did consider a visual cue but decided it wasn't necessary since we only allow popularity transfers if it *improves* the search results. If a customer were to complain that search results are unexpected, this would be a strong signal that we messed up and allowed a popularity transfer that regresses search rankings.

/cc @chgill-MSFT who drove the spec for popularity transfers

Reply:
Loic nailed the explanation exactly. The value of this feature is to actively _improve_ search results in cases where search ranking prioritizing downloads works against the best interest of our users. If we hear a lot of negative customer feedback or see in telemetry that search has been negatively impacted - we'll iterate on the experience accordingly 🙂

Reply:
Loic nailed the explanation exactly. The value of this feature is to actively _improve_ search results in cases where search ranking prioritizing downloads works against the best interest of our users. If we hear a lot of negative customer feedback or see in telemetry that search has been negatively impacted - we'll iterate on the experience accordingly 🙂